spoilers/frontend-step-04-extra-credit.md

```diff
@@ -3,7 +3,6 @@ if (
   selectedAlternateShippingInstruction !== 'other' ||
   otherShippingValue !== ''
 ) {
-  clearValidationError( validationErrorId );
```

Comment:
Curious why this was removed? If there is a validation error first, then the value is changed away from `other` checkout would be prevented without this line, wouldn't it?

Reply:
Yes, you are right. I was trying to figure out how to align spoilers with tasks and I overdid here. Maybe, the [frontend-step-04-extra-credit] could be removed? I see how it can be implement through wrapping `setValidationErrors` in an `if`, but it goes way far from the following logic of steps 05 and 07.

Reply:
ah I see, so because it's covered in extra-credit, we want to remove it to allow implementation, I'm fine with that.